moveit_core/collision_distance_field/src/collision_world_distance_field.cpp

```diff
@@ -472,8 +479,8 @@ void CollisionWorldDistanceField::setWorld(const WorldPtr& world)
 void CollisionWorldDistanceField::notifyObjectChange(CollisionWorldDistanceField* self, const ObjectConstPtr& obj,
                                                      World::Action action)
 {
-  ros::WallTime n = ros::WallTime::now();
-
+  // WallTime
+  auto n = std::chrono::system_clock::now();
```

Comment:
http://docs.ros2.org/bouncy/api/rclcpp/classrclcpp_1_1_clock.html

You can use ros::Clock with RCL_SYSTEM_TIME

Reply:
https://github.com/ros-planning/moveit2/pull/65/commits/7f4b0e5a4f3a7fd110f213daeac085fb11294607